examples-sdk-s3/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright 2018-2019 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>atum-examples-sdk-s3_2.11</artifactId>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>za.co.absa</groupId>
+        <artifactId>atum-parent_2.11</artifactId>
+        <version>3.2.1-SNAPSHOT</version>
+    </parent>
+
+    <dependencies>
+        <!-- Atum -->
+        <dependency>
```

Comment:
This is out of alignment

Reply:
Sorry, I don't see the problem. What exactly seems unaligned?

Reply:
Use spaces instead of tabs. If you have tabs set to something different then I, then it looks good on your screen.

Reply:
Oh, I see. Thanks. Done.